content/patterns/checkbox/checkbox-pattern.html

```diff
@@ -14,61 +14,62 @@
-          And, each option in the group can be individually turned on or off with a dual state checkbox.
+          三つの状態のチェックボックスの一般的な使用方法の1つは、ソフトウェアのインストーラです。三つの状態のチェックボックスがインストールオプションのグループ全体の状態を表し、制御するために使用されます。
+          そして、グループ内の各オプションは、二つの状態のチェックボックスで個別にオン又はオフにすることができます。
         </p>
         <ul>
-          <li>If all options in the group are checked, the overall state is represented by the tri-state checkbox displaying as checked.</li>
-          <li>If some of the options in the group are checked, the overall state is represented with the tri-state checkbox displaying as partially checked.</li>
-          <li>If none of the options in the group are checked, the overall state of the group is represented with the tri-state checkbox displaying as not checked.</li>
+          <li>グループ内のすべてのオプションがチェックされている場合、全体の状態は、三つの状態のチェックボックスがチェックされることで表現されます。</li>
+          <li>グループ内の一部のオプションがチェックされている場合、全体の状態は、三つの状態のチェックボックスの一部がチェックされることで表現されます。</li>
+          <li>グループ内のオプションが1つもチェックされていない場合、全体の状態は、三つの状態のチェックボックスがチェックされていないことで表現されます。</li>
         </ul>
-        <p>The user can use the tri-state checkbox to change all options in the group with a single action:</p>
+        <p>ユーザーは、三つの状態のチェックボックスを使用して、グループ内のすべてのオプションを1つのアクションで変更することができます：</p>
         <ul>
-          <li>Checking the overall checkbox checks all options in the group.</li>
-          <li>Unchecking the overall checkbox will uncheck all options in the group.</li>
+          <li>全体のチェックボックスをチェックすると、グループ内のすべてのオプションがチェックされます。</li>
+          <li>全体のチェックボックスのチェックを外すと、グループ内のすべてのオプションのチェックが外れます。</li>
           <li>
-            And, In some implementations, the system may remember which options were checked the last time the overall status was partially checked.
-            If this feature is provided, activating the overall checkbox a third time recreates that partially checked state where only some options in the group are checked.
+            そして、一部の実装では、最後にどのオプションがチェックされていたかを記憶する場合があります。
+            この機能が提供されている場合、全体のチェックボックスを3回目に作動させると、グループ内の一部のオプションのみがチェックされている状態が再現されます。
           </li>
         </ul>
       </section>
 
       <section id="examples">
-        <h2>Examples</h2>
+        <h2>例</h2>
         <ul>
-          <li><a href="examples/checkbox.html">Checkbox (Two-State) Example</a>: Demonstrates a simple 2-state checkbox.</li>
-          <li><a href="examples/checkbox-mixed.html">Checkbox (Mixed-State) Example</a>: Demonstrates a checkbox that uses the mixed value for aria-checked to reflect and control checked states within a group of two-state HTML checkboxes contained in an HTML <code>fieldset</code>.</li>
+          <li><a href="examples/checkbox.html">チェックボックス（二つの状態）の例</a>：シンプルな二つの状態のチェックボックスのデモンストレーションです。</li>
+          <li><a href="examples/checkbox-mixed.html">チェックボックス（三つの状態）の例</a> ：HTML <code>fieldset</code> 内に含まれる二つの状態の HTML チェックボックスのグループ内のチェック状態を反映し制御するために、aria-checked の混合値を使用するチェックボックスのデモンストレーションです。</li>
```

Comment:
mixed valueは aria-checked="mixed" という指定のことを指しているので、そのままmixedで良いかと。「aria-checked に mixed を指定」というくらいでしょうか。

```suggestion
          <li><a href="examples/checkbox-mixed.html">チェックボックス（三状態）の例</a> ：HTML <code>fieldset</code> 内に含まれる二状態の HTML チェックボックスのグループ内のチェック状態を反映し制御するために、aria-checked に mixed という値を指定するチェックボックスのデモンストレーションです。</li>
```